### ISSUE TITLE
【Feature】can i use ch_PP-OCRv3_rec_distillation.yml to distill official en_PP-OCRv3_rec infrer model?

### ISSUE THREAD

**Issue by PancakeAwesome** [PaddlePaddle/PaddleOCR#10388]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
can i use ch_PP-OCRv3_rec_distillation.yml to distill official en_PP-OCRv3_rec infrer model?
or how can i finetune official en ppocrv3 model with distill (or not )in my field dataset, and did well?
thank you ~
- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

**Comment by PancakeAwesome** [comment]
additional, my dataset is an LCD number, include 0-9 and . ten labels totally. such as :
https://p.ipic.vip/rxfxlb.jpeg

**Comment by GreatV** [comment]
it is OK to load the official **pre-trained** model and fine-tune it in your own data set. please refer to: [Model Fine-tuning Tutorial](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/finetune_en.md).

**Comment by PancakeAwesome** [comment]
> it is OK to load the official **pre-trained** model and fine-tune it in your own data set. please refer to: [Model Fine-tuning Tutorial](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/finetune_en.md).

Thank you for your reply.

- And I wanna know can i tunning official en rec ppocrv3 train model with offical distill config file, like:configs/rec/PP-OCRv3/ch_PP-OCRv3_rec_distillation.yml?Cause i just found one distill rec yml config file,like up.
- Or i want to know what is best practice for en rec model tunning? distill with official yml or just finetune by loading rec model with en yml,like configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml
Look forward to your rely~

**Comment by GreatV** [comment]
> can i tunning official en rec ppocrv3 train model with offical distill config file.

Yes

> en rec ppocr pretrained model 

[English Recognition Model](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/models_list_en.md#22-english-recognition-model)

just finetune by loading rec model with en yml work well
